robot/web/web.go

```diff
@@ -55,120 +50,6 @@ var API = resource.APINamespaceRDKInternal.WithServiceType(SubtypeName)
 // InternalServiceName is used to refer to/depend on this service internally.
 var InternalServiceName = resource.NewName(API, "builtin")
 
-// robotWebApp hosts a web server to interact with a robot in addition to hosting
-// a gRPC/REST server.
-type robotWebApp struct {
-	template *template.Template
-	theRobot robot.Robot
-	logger   logging.Logger
-	options  weboptions.Options
-}
-
-// Init does template initialization work.
-func (app *robotWebApp) Init() error {
-	var err error
-
-	t := template.New("foo").Funcs(template.FuncMap{
-		//nolint:gosec
-		"jsSafe": func(js string) template.JS {
-			return template.JS(js)
-		},
-		//nolint:gosec
-		"htmlSafe": func(html string) template.HTML {
-			return template.HTML(html)
-		},
-	}).Funcs(sprig.FuncMap())
-
-	if app.options.SharedDir != "" {
-		t, err = t.ParseGlob(fmt.Sprintf("%s/*.html", app.options.SharedDir+"/templates"))
-	} else {
-		t, err = t.ParseFS(web.AppFS, "runtime-shared/templates/*.html")
-	}
-
-	if err != nil {
-		return err
-	}
-	app.template = t.Lookup("webappindex.html")
-	return nil
-}
-
-// AppTemplateData is used to render the remote control page.
-type AppTemplateData struct {
-	WebRTCEnabled          bool                   `json:"webrtc_enabled"`
-	WebRTCSignalingAddress string                 `json:"webrtc_signaling_address"`
-	Env                    string                 `json:"env"`
-	Host                   string                 `json:"host"`
-	StaticHost             string                 `json:"static_host"`
-	SupportedAuthTypes     []string               `json:"supported_auth_types"`
-	AuthEntity             string                 `json:"auth_entity"`
-	BakedAuth              map[string]interface{} `json:"baked_auth"`
-}
```

Comment:
can new local app handle no credentials passed, insecure, like the one you're deleting did?

I can not run rdk without credentials and use the sdks to connect to it with an insecure option, if this possible in the new world where we build "local" rc?

Reply:
Insecure connections haven't been built into the svelte-sdk. I can add them easily if it would be useful to you

Reply:
This is what @dgottlieb is looking for as well - we need a small TS SDK change in order to provide this from the `svelte-sdk` (aka the new local control page). Opened [here](https://github.com/viamrobotics/viam-typescript-sdk/pull/411), and will also open the corresponding change against the `svelte-sdk`.